codejam-virtual-keyboard/src/components/Keyboard/Keyboard.scss

```diff
@@ -0,0 +1,55 @@
+$keyboard-layer-color: silver;
```

Comment:
FYI: `silver` in Chrome can be not the same color like in another browser, thus it is better to set it as `#aaa9ad` or using `rgb, rgba`.